test/values.ts

```diff
@@ -1668,6 +1668,95 @@ export const mockPositions: Position[] = [
   },
 ]
 
+export const mockRewardsPositions: Position[] = [
+  {
```

Comment:
optional nit: can we reuse `mockEarnPositions` for the first one here?

Reply:
I was trying to keep the amount of unit test changes minimal by using the same values that were in the test data before so that Its obvious to a reader that its a refactor.

Reply:
I can clean these mock datas up in a followup pr